Sources/SwifterSwift/CoreGraphics/CGSizeExtensions.swift

```diff
@@ -14,7 +14,8 @@ public extension CGSize {
 
     /// SwifterSwift: Returns the aspect ratio.
     var aspectRatio: CGFloat {
-        return height == 0 ? 0 : width / height
```

Comment:
So this is one of the compile time optimisations. The ternary operator `?:` for some reason is really slow to compile. The one-liner takes ~250ms (on my machine) to compile, but the 2-liner is about 30ms. I know it's not exactly a massive optimisation, but it's still a saving and interesting to learn that this happens in the Swift compiler.

Reply:
Type checking is done in an expression by expression basis, I think the only thing is happening here is that the time is being divided by two expressions instead of one... not sure tho, maybe it is also saving a few overload choice attempts because the two operators aren't in a single expression, but I think is not because width and height types are known upfront. 
But I think this is ok

Reply:
Xcode complains about the type-checking time for the entire function, not each line. I've read a few blogs saying that they discovered the same problem.